### ISSUE TITLE
Unable to share one password among users

### ISSUE THREAD

**Issue by ScreamingDev** [nextcloud/passman#388]
### Steps to reproduce
1. Create a password
2. Create a second user
3. Share password among users

### Expected behaviour

Other user has the password in the list.
Editing will be reflected in the others list too.

### Actual behaviour

Unable to share. Button still greyed out after selecting a user.

![image](https://user-images.githubusercontent.com/2559177/32355229-d1b9f98a-c02d-11e7-9850-f3580deee68d.png)

### Configuration
**Operating system**:   

Ubuntu 16.04

**Browser**:      

Chrome

**Extensions that might cause interference**:   

Got "link share" disabled.

**Passman version**:    

2.1.4

**Operating system**:

Still Ubuntu.

**Web server:**

Apache

**Database:**

MariaDB

**PHP version:**

7.0.22
      
**cloud server:** Nextcloud 12

**Comment by brantje** [comment]
Does the second user has a vault? Maybe @animalillo can help us out here =)

**Comment by ScreamingDev** [comment]
Yes, both have their own vault. Did you check the behavior (on the demo server or some other) ?
